Assets/Package/Runtime/Views/ViewsHandler.cs

```diff
@@ -0,0 +1,265 @@
+﻿using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace BroWar.UI.Views
+{
+    using BroWar.Common;
+
+    /// <inheritdoc cref="IUiViewsHandler"/>
+    [DisallowMultipleComponent]
+    [AddComponentMenu("BroWar/UI/Views/Views Handler")]
+    public class ViewsHandler : UiHandlerBehaviour, IUiViewsHandler
+    {
+        protected readonly Dictionary<Type, UiView> viewsByTypes = new Dictionary<Type, UiView>();
+        protected readonly List<UiView> activeViews = new List<UiView>();
+
+        [SerializeField]
+        [Tooltip("Indicates if handler should be initialized during the 'Prepare' callback.")]
+        protected bool selfInitialize = true;
```

Comment:
Może w takim razie initializeOnPrepare? Tak jak mamy wszędzie initializeOnStart?

Reply:
Zostawiłbym tak jak jest. Nawet skłaniam się żeby raczej używać selfInitialize bo jest to bardziej generyczna nazwa, a jak istnieje potrzeba doprecyzowania co to za moment to można zrobić to w tooltipie